### DISCUSSION TITLE
How can I use fonttools to easily install a font to a users system?

### DISCUSSION THREAD

**Discussion by Moosems** [fonttools/fonttools#3203] [General]
I would like to install a ttf file to the users system for an application using fonttools and it seems like it should be possible but I'm unsure how this would be done.

**Comment by Moosems** [comment]
The save() method can copy it to any location but is there a method that finds the users font dir and saves the file there?

**Reply by justvanrossum** [reply]
This is out of scope for fonttools.

**Reply by Moosems** [reply]
So what is within the scope of fonttools?

**Reply by behdad** [reply]
FontTools can manipulate font files, in isolation from where they are installed.

**Reply by justvanrossum** [reply]
The one-liner description of the project is "A library to manipulate font files from Python". We should probably add "and create" after "manipulate". Creating and modifying fonts. 

Font installation is something that varies wildly depending on deployment context (mostly: platform) and is not something fonttools aims to help with.
